stable/mongodb/values.yaml

```diff
@@ -187,6 +187,11 @@ persistence:
   size: 8Gi
   annotations: {}
 
+  extraInitContainers: |
```

Comment:
Could you please add a previous comment introducing the purpose of this parameter? Please also add a reference to https://kubernetes.io/docs/concepts/workloads/pods/init-containers/

Reply:
Please also comment this line to avoid issues with `{{- if .Values.extraInitContainers }}`

Reply:
@juan131 Done!